BrainPortal/lib/scir_cloud.rb

```diff
@@ -52,8 +52,328 @@ def self.get_available_disk_images(bourreau)
   #     [ ["id_rsa_cbrain_portal"], ["personal_key"] ]
   # This (weird) data structure is used to pass the result of this method in a Rails select tag.
   def self.get_available_key_pairs(bourreau)
-    raise "Not implemented"
+    raise "Needs to be implemented in a sub-class"
+  end
+  
+  # Returns true of task is a VM
+  def self.is_vm_task?(task)
+    return task.is_a?(CbrainTask::StartVM)
```

Comment:
I dislike a little that this link the Scir class to the CbrainTask class, but ok. One day we'll have to make sure all these layers are not interdependent.